src/main/java/application/rest/v1/KAppNavEndpoint.java

```diff
@@ -443,25 +468,29 @@ private static String getErrorMessageAsJSON(String msg) {
     // Generic CRUD APIs 
 
     protected Object getNamespacedGenericObject(ApiClient client, String group, String kindPlural, String namespace, String name) throws ApiException {
+        Logger.log(className, "getNamespacedGenericObject", Logger.LogType.INFO, "For group=" + group + ", kindPlural="+kindPlural + ", namespace="+namespace + ", name="+name);
```

Comment:
Another set of INFOs that look like they should be DEBUG.

Reply:
I changed it to use DEBUG.